src/client/components/pages/search.js

```diff
@@ -116,46 +97,14 @@ class SearchPage extends React.Component {
 					onSearch={this.handleSearch}
 				/>
 				<SearchResults results={this.state.results}/>
-				{
-					this.state.results && this.state.results.length ?
-						<div>
-							<hr className="thin"/>
-							<Pager>
-								<Pager.Item
-									previous disabled={this.state.from <= 0}
-									href="#" onClick={this.handleClickPrevious}
-								>
-									&larr; Previous Page
-								</Pager.Item>
-								<ButtonGroup>
-									<Button disabled>Results {this.state.from + 1} —
-										{this.state.results.length < this.state.size ?
-											this.state.from + this.state.results.length :
-											this.state.from + this.state.size
-										}
-									</Button>
-									<DropdownButton
-										dropup bsStyle="info" id="bg-nested-dropdown"
-										title={`${this.state.size} per page`}
-										onSelect={this.handleResultsPerPageChange}
-									>
-										<MenuItem eventKey="10">10 per page</MenuItem>
-										<MenuItem eventKey="20">20 per page</MenuItem>
-										<MenuItem eventKey="35">35 per page</MenuItem>
-										<MenuItem eventKey="50">50 per page</MenuItem>
-										<MenuItem eventKey="100">100 per page</MenuItem>
-									</DropdownButton>
-								</ButtonGroup>
-								<Pager.Item
-									next disabled={!this.state.nextEnabled}
-									href="#" onClick={this.handleClickNext}
-								>
-									Next Page &rarr;
-								</Pager.Item>
-							</Pager>
-						</div> :
-						null
-				}
+				<PagerElement
+					changeDataInTable={this.changeDataInTable}
```

Comment:
This name suggests to me that calling it will change the data.
Instead it's a callback; I would suggest another name to reflect that (everywhere it is used): `searchResultsCallback`

Reply:
Yeah I didn't give much thought while naming the function. Thanks :D

Reply:
It's always a tricky exercise, naming variables and functions !